features/confidential-transactions/index.md

```diff
@@ -88,15 +88,13 @@ elements-cli sendrawtransaction $TX
 
 ### Limitations
 
-The implementation of Confidential Transactions as it appears in Elements has some important limitations to be aware of.
+The implementation of Confidential Transactions, as it appears in Elements, has some important limitations to be aware of.
 
-The implementation only hides a certain number of the digits of the amount of each transaction output, dependent on the range proof's "blinding coverage" at a desired precision level.  Subsequently, there is a "minimum confidential amount" around 0.0001, and a "maximum confidential amount" that is 2<sup>32</sup> times the minimum amount.
+The implementation only hides a certain number of the digits of the amount of each transaction output, dependent on the range proof's "blinding coverage" at a desired precision level. The default blinding precision is currently 52 bits, which will cover any value up to 2^52 satoshi, which equates to a maximum amount of just over 45 million. As the maximum amount that can be transacted is inherited from Bitcoin, and set at 21 million, this currently provides cover for all asset transactions.
```

Comment:
> any value up to 2^52 satoshi, which equates to a maximum amount of just over 45 million

45 million BTC*

Reply:
Lateron also 21 million BTC*.

Reply:
The amount is not necessary in BTC, it may be some other asset. Maybe "X million coins" ?

Reply:
...or X million BTC equivalent units.

Bit wordy though.